package.json

```diff
@@ -138,7 +139,7 @@
     "yuidocjs": "0.10.2"
   },
   "engines": {
-    "node": ">= 0.10.0"
+    "node": "0.12.* || 4.5.* || 6.*"
```

Comment:
The 4.5.* strikes me as odd.  In theory, there could well be a `4.6` that would be part of the 4.x LTS series.

From https://github.com/nodejs/LTS#lts-plan:

> Note that while it is possible that critical security and bug fixes may lead to semver-major changes landing within an LTS stream, such situations will be rare and will land as semver-minor bumps in the LTS covered version.

Is your thought that these "major" bumps from critical bugfixes are not supported?  It seems like that would be unlikely...